mkosi

```diff
@@ -1014,7 +1018,7 @@ def mount_cache(args: CommandLineArguments, workspace: str) -> Generator[None, N
 
 def umount(where: str) -> None:
     # Ignore failures and error messages
-    run(["umount", "--recursive", "-n", where], stdout=DEVNULL, stderr=DEVNULL)
+    run(["umount", "-n", where], stdout=DEVNULL, stderr=DEVNULL)
```

Comment:
What about instead of doing this turning this into a mountpoint in the squashfs case and avoid the problem here?
`mount --bind root root` should work and then the `umount --recursive`, too.

Reply:
yeah, i think this is a nice approach and makes our codepaths a bit more alike